packages/blade/src/components/Box/Box.d.ts

```diff
@@ -24,11 +25,19 @@ export type BoxProps = {
   marginBottom?: DotNotationSpacingStringToken | 'auto' | number;
   marginLeft?: DotNotationSpacingStringToken | 'auto' | number;
   marginRight?: DotNotationSpacingStringToken | 'auto' | number;
+  width?: CSSObject['width'];
   gap?: DotNotationSpacingStringToken | 'auto' | number;
   minHeight?: number;
   minWidth?: number;
   maxHeight?: number;
   maxWidth?: number;
+  position?: CSSObject['position'];
+  transform?: CSSObject['transform'];
+  top?: CSSObject['top'];
+  left?: CSSObject['left'];
+  right?: CSSObject['right'];
+  bottom?: CSSObject['bottom'];
+  zIndex?: CSSObject['zIndex'];
```

Comment:
You changed the layout logic.. just see if this are extraneous or needed.

Reply:
fixed in e263fd9b727b4b26f7de680d3ce34de705cca296